README.md

```diff
@@ -1,2 +1,55 @@
 # hyper-sdk-expo-plugin
-Repository for Expo support on HyperSDK React
+
+An Expo config plugin for automating Android & iOS integration changes for [`hyper-sdk-react`](https://www.npmjs.com/package/hyper-sdk-react) with apps using Expo.
+
+## Installation
+
+```bash
+npm install expo-hyper-sdk-plugin
```

Comment:
In this step, we are calling it `expo-hyper-sdk-plugin` while below, we are calling it `hyper-sdk-expo-plugin`. Can we keep it uniform (`hyper-sdk-expo-plugin`) ?